nusantara/nusa_datasets/id_short_answer_grading/id_short_answer_grading.py

```diff
@@ -0,0 +1,194 @@
+                English short answer grading dataset with 80 questions and 2442 to get the best configuration for model and Indonesian short answer grading
+                dataset with 36 questions and 9165 short answers as testing data. Here, we’ve collected Indonesian short answers for Biology and Geography
+                subjects from 534 respondents where the answer grading was done by 7 experts. The best root mean squared error for both dataset was achieved
+                by using BERT pretrained, 0.880 for English dataset dan 1.893 for Indonesian dataset.},
+    issn = {2621-9336},	pages = {54--61},	doi = {10.26418/jlk.v3i2.38},
+    url = {https://inacl.id/journal/index.php/jlk/article/view/38}
+}\
+"""
+_DATASETNAME = "id_short_answer_grading"
+
+_DESCRIPTION = """\
+Indonesian short answers for Biology and Geography subjects from 534 respondents where the answer grading was done by 7 experts.\
+"""
+
+_HOMEPAGE = ""
+
+_LICENSE = ""
+
+_URLS = {
+    "saintek": {
+        "train": {
+            "question": "https://raw.githubusercontent.com/AgeMagi/tugas-akhir/master/data/question-saintek.csv",
+            "score": "https://raw.githubusercontent.com/AgeMagi/tugas-akhir/master/data/score-saintek.csv",
+        },
+        "test": {
+            "question": "https://raw.githubusercontent.com/AgeMagi/tugas-akhir/master/data/question-saintek-test.csv",
+            "score": "https://raw.githubusercontent.com/AgeMagi/tugas-akhir/master/data/score-saintek-test.csv",
+        },
+    },
+    "soshum": {
+        "train": {
+            "question": "https://raw.githubusercontent.com/AgeMagi/tugas-akhir/master/data/question-soshum.csv",
+            "score": "https://raw.githubusercontent.com/AgeMagi/tugas-akhir/master/data/score-soshum.csv",
+        },
+        "test": {
+            "question": "https://raw.githubusercontent.com/AgeMagi/tugas-akhir/master/data/question-soshum-test.csv",
+            "score": "https://raw.githubusercontent.com/AgeMagi/tugas-akhir/master/data/score-soshum-test.csv",
+        },
+    },
+}
+
+_SUPPORTED_TASKS = [Tasks.SHORT_ANSWER_GRADING]
+
+_SOURCE_VERSION = "1.0.0"
+
+_NUSANTARA_VERSION = "1.0.0"
+
+
+class IdShortAnswerGrading(datasets.GeneratorBasedBuilder):
+    """Indonesian short answers for Biology and Geography subjects from 534 respondents where the answer grading was done by 7 experts."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    NUSANTARA_VERSION = datasets.Version(_NUSANTARA_VERSION)
+
+    BUILDER_CONFIGS = [
+        NusantaraConfig(
+            name="id_short_answer_grading_source",
+            version=SOURCE_VERSION,
+            description="id_short_answer_grading source schema",
+            schema="source",
+            subset_id="id_short_answer_grading",
+        ),
+        NusantaraConfig(
+            name="id_short_answer_grading_nusantara_pairs_score",
```

Comment:
Since the label is in [0, 1, 2, 3, 4, 5], I think we can use `pairs_features` rather than `pairs_features_score`. Note that the `pairs_features_score` is an ad-hoc schema used for dataset that cannot be converted into a discrete label.

Could you please change the schema to `nusantara_pairs` instead of `nusantara_pairs_score` and update the features accordingly?

Reply:
okay kaa, i'm already on it. although i keep failing a test which says
```
Traceback (most recent call last):
  File "D:\nusa-crowd\tests\test_nusantara.py", line 129, in runTest
    self.test_schema(schema)
  File "D:\nusa-crowd\tests\test_nusantara.py", line 553, in test_schema
    self.assertEqual(type(features[key]), type(split.info.features[key]))
AssertionError: <class 'datasets.features.features.Value'> != <class 'datasets.features.features.ClassLabel'>
```
after changing the schema to `schemas.pairs_features([0, 1, 2, 3, 4, 5])` from `schemas.pairs_features_score()`.

however, after tinkering for a while, i remembered that the `Tasks.SHORT_ANSWER_GRADING` is new and so in `test_nusantara.py`, i tried changing `Tasks.SHORT_ANSWER_GRADING: "PAIRS_SCORE"` to `Tasks.SHORT_ANSWER_GRADING: "PAIRS"` which as it turns out, solves it. I wonder if it's the right thing to do? (I know changing tests feels like cheating) but if it is, please let me know if I should include the change in this commit or if I should just wait for it to be changed (it being a new task and done by someone else) ***or*** if I should do something else completely different than changing `test_nusantara.py`

thanks in advance kaa

Reply:
I think it's the correct thing to do

Reply:
i see, so i take it that i should include the change in this commit, then, kak?